Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -702,7 +723,7 @@ protected function getFlattenedValidationErrorMessage()
                 $logMessage .= 'Error for ' . $propertyPath . ':  ' . $error->render() . PHP_EOL;
             }
         }
-        $this->systemLogger->log($logMessage, LOG_ERR);
+        $this->logger->log(LogLevel::ERROR, $logMessage);
```

Comment:
Why not `logger->error()`?

Reply:
It was easier to search replace to this format 😆